intel_extension_for_transformers/transformers/utils/utility.py

```diff
@@ -30,6 +30,7 @@
 DECODER_WITH_PAST_NAME = "decoder_with_past_model.bin"
 WEIGHTS_NAME = "pytorch_model.bin"
 
+torch = LazyImport("torch")
```

Comment:
better to rename the file to "generation_utility.py"?

Reply:
it seems not only have generate utility but also have dataloader and distribute utility. @hshen14 @PenghuiCheng